src/components/ProvisionButton/index.tsx

```diff
@@ -48,6 +48,7 @@ class ProvisionButton extends React.Component<IProvisionButtonProps, IProvisionB
         firewallStartIPAddress: "0.0.0.0",
         location: "eastus",
         resourceGroup: "default",
+        sslEnforcement: "disabled",
```

Comment:
Added `sslEnforcement: "disabled"` to default json to ensure app can connect to the db instance ootb.